README.md

```diff
@@ -19,28 +19,28 @@ month,observation
 
 These are the statistics that are fetched, reformatted and stored in the `./data` directory:
 - Argentina
-  - [Consumer price index – monthly year-on-year](https://datos.gob.ar/series/api/series/?ids=148.3_INIVELNAL_DICI_M_26&collapse=month&collapse_aggregation=avg&representation_mode=percent_change_a_year_ago)  
+  - [Consumer price index – monthly year-on-year](https://fullfact.github.io/nso-stats-fetcher/data/ar_inflation_cpi.csv) ([source](https://datos.gob.ar/series/api/series/?ids=148.3_INIVELNAL_DICI_M_26&collapse=month&collapse_aggregation=avg&representation_mode=percent_change_a_year_ago))
 - Ireland
-  - [Consumer price index – monthly year-on-year](https://data.cso.ie/product/CPIM)
+  - [Consumer price index – monthly year-on-year](https://fullfact.github.io/nso-stats-fetcher/data/ie_inflation_cpi.csv) ([source](https://data.cso.ie/product/CPIM))
 - Japan
-  - [Consumer price index – monthly year-on-year](https://www.stat.go.jp/english/data/cpi/1581-z.html)
+  - [Consumer price index – monthly year-on-year](https://fullfact.github.io/nso-stats-fetcher/data/jp_inflation_cpi.csv) ([source](https://www.stat.go.jp/english/data/cpi/1581-z.html))
 - Mexico
-  - [Consumer price index – monthly year-on-year](https://www.stat.go.jp/english/data/cpi/1581-z.html)
+  - [Consumer price index – monthly year-on-year](https://fullfact.github.io/nso-stats-fetcher/data/mx_inflation_cpi.csv) ([source](https://www.stat.go.jp/english/data/cpi/1581-z.html))
```

Comment:
Looks like the wrong URL got copied here as source.

Reply:
Oh, you’re right – though that’s not changed by this PR. In any case, I’ll add a fix for it and will check the rest.

Reply:
Hopefully fixed in 943eb2f.